cobra/test/test_model.py

```diff
@@ -691,6 +691,33 @@ def test_add_reaction_orphans(self, model):
         # 'check not dangling metabolites when running Model.add_reactions
         assert len(orphan_metabolites) == 0
 
+    def test_merge_models(self, model, tiny_toy_model):
+        with model, tiny_toy_model:
```

Comment:
That all came for free as using `add_cons_vars` etc and the context in the unit-tests was mainly so that we don't have re-load the model for the next test, but true, a couple of extra checks doesn't hurt; added